tests/test_pipeline/test_pipeline.py

```diff
@@ -597,7 +775,7 @@ def test_generate_folds_datasets(ts_name, mask, request):
     """Check _generate_folds_datasets for correct work."""
     ts = request.getfixturevalue(ts_name)
     pipeline = Pipeline(model=NaiveModel(lag=7))
-    mask = pipeline._prepare_fold_masks(ts=ts, masks=[mask], mode="constant")[0]
+    mask = pipeline._prepare_fold_masks(ts=ts, masks=[mask], mode="constant", stride=-1)[0]
```

Comment:
Is it ok that stride is negative?

Reply:
Method `_prepare_fold_masks` ignores `stride if `FoldMasks` are given. I don't really sure what value should we pass here during the test. I put `-1` to indicate that it doesn't make sense and has no influence.